app/components/category-form/component.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+import { categoryValidations } from 'bsc-ember/validations/category';
+import Changeset from 'ember-changeset';
+import lookupValidator from 'ember-changeset-validations';
+
+const { Component,
+  inject: {
+    service
+  } } = Ember;
+
+export default Component.extend({
+  router: service(),
+  init() {
+    this._super(...arguments);
+    this.changeset = new Changeset(this.model, lookupValidator(categoryValidations), categoryValidations);
```

Comment:
it's might be better if we will do this on `didReceiveAttrs`

Do you know why?